pkg/security/probe/event_stats.go

```diff
@@ -10,8 +10,10 @@ package probe
 import "sync/atomic"
 
 // EventsStats holds statistics about the number of lost and received events
+//nolint:structcheck,unused
 type EventsStats struct {
 	Lost         int64
+	padding      int32
```

Comment:
Why is this `padding` needed? Shouldn't this already be 64 bit aligned because `Lost` is 64 bits.

Reply:
That's what I expected but it segfaults without the padding :-/

Reply:
😭